lib/formatter.js

```diff
@@ -1,11 +1,21 @@
-const contributorsReport = (contributors) => (
-  Object.entries(contributors || {}).reduce((acc, [login, { commits, fresh }], i) => {
-    acc += `${i === 0 ? '\n\n# Contributors: \n' : ''}\n- [@${login}](https://github.com/${login}) (${fresh ? '🎉 fresh contributor, ' : ''}${commits} commits)`
-    return acc
-  }, '')
-)
+const contributorsReport = ({ contributors, since }) => {
+  if (!Object.keys(contributors).length) {
+    return ''
+  }
+  let r = '\n\n# Contributors:\n'
+  Object.entries(contributors || {}).forEach(([login, { commits, fresh, days } ]) => {
+    r += `\n- [@${login}](https://github.com/${login})`
+    if (fresh) {
+      r += ` (🎉 fresh contributor with ${commits} commits in ${days.size} day${days > 1 ? 's' : ''} since at least ${since.split('T')[0]})`
```

Comment:
the wording is slightly confusing, because `x days` makes me think of a date range instead of just number of days. 

However, I can't really think of a better way to say it.

Reply:
we can potentially clarify by:

> fresh contributor with 3 commits (made in 1 day) since at least 2021-12-31

tho these stats could also be argued as not as important 🤔

Reply:
yeah I don't think it's a big deal

Reply:
it's revised in the latest commit, would be in the fashion of

```
- [@woozyking](https://github.com/woozyking) (returning contributor - 11 commits across 2 days)
```